wren-modeling-py/pyproject.toml

```diff
@@ -17,6 +17,7 @@ pytest = "8.3.3"
 
 [tool.maturin]
 module-name = "wren_core"
+profile = "release"
```

Comment:
Curious, what does this do?

Reply:
The `release` profile will optimize the binary more.